lib/blocks.php

```diff
@@ -78,6 +79,26 @@ function gutenberg_reregister_core_block_types() {
 }
 add_action( 'init', 'gutenberg_reregister_core_block_types' );
 
+/**
+ * Adds new block categories needed by the Gutenberg plugin.
+ *
+ * @param array $categories List of block categories.
+ *
+ * @return array List of block categories with the new categories added.
+ */
+function gutenberg_filter_block_categories( $categories ) {
+	return array_merge(
+		$categories,
+		array(
+			array(
+				'slug'  => 'theme',
+				'title' => __( 'Theme Blocks', 'gutenberg' ),
```

Comment:
Why call the category "Theme Blocks"? The blocks aren't added by the theme. In fact, they're not really related to the theme at all. Wouldn't something like "Site Blocks" be more appropriate? "Theme Blocks" seems misleading.

Reply:
What about `Template Blocks`? 😄

Reply:
I think `Template` is a term that developers, not users, are more familiar with, but I also think that `Theme` could be misinterpreted as blocks provided by a theme or as simply blocks made for styling.

`Template` might not be the right term though, as these blocks can also be used inside regular posts.

I think `Site` makes the most sense here, with `Theme` being a close second.

Reply:
There's some ongoing conversions here about updating the block categories to be more meaningful: https://github.com/WordPress/gutenberg/issues/11406

Reply:
Could we keep these within the existing Layout Elements group, rather than adding another group with more jargon?

At some point, we'll regroup all of these. I'd also expect to see some guidelines around grouping blocks as the Block Directory comes about and more singular blocks are available.

Reply:
I'd prefer an existing category as well until we can improve it more generally.

Reply:
That sounds good, I actually originally put them in `Layout Elements`.

Reply:
Changed to `Layout Elements` ✅